frontend/src/scenes/insights/common.tsx

```diff
@@ -2,10 +2,16 @@ import React from 'react'
 import { InfoCircleOutlined } from '@ant-design/icons'
 import { Tooltip } from 'lib/components/Tooltip'
 
-export function GlobalFiltersTitle({ unit = 'series' }: { unit?: string }): JSX.Element {
+export function GlobalFiltersTitle({
+    unit = 'series',
+    title = 'Filters',
```

Comment:
It felt wrong to use this for Exclusions, since the tooltip text has to change as well 😅 . Most other places I checked didn't try to reuse this (although, I can very well see the case for abstracting it out everywhere).

But at the same time, it's such a small component that I just rewrote what we needed. Upto you to decide where to go with this one :)